tap_snowflake/__init__.py

```diff
@@ -321,6 +328,11 @@ def resolve_catalog(discovered_catalog, streams_to_sync):
         # These are the columns we need to select
         columns = desired_columns(selected, discovered_table.schema)
 
+        if (len(columns) > MAX_COLS):
+            raise Exception(f'Number of columns cannot exceed {MAX_COLS} ')
```

Comment:
Can we get rid of the trailing space too?

Reply:
done